xds/src/main/java/io/grpc/xds/XdsLoadBalancerProvider.java

```diff
@@ -78,8 +76,38 @@ static ConfigOrError parseLoadBalancingConfigPolicy(
       Map<String, ?> rawLoadBalancingPolicyConfig, LoadBalancerRegistry registry) {
     try {
       String cluster = JsonUtil.getString(rawLoadBalancingPolicyConfig, "cluster");
-      LbConfig childPolicy = selectChildPolicy(rawLoadBalancingPolicyConfig, registry);
-      LbConfig fallbackPolicy = selectFallbackPolicy(rawLoadBalancingPolicyConfig, registry);
+
+      LbConfig roundRobinConfig = new LbConfig("round_robin", ImmutableMap.<String, Object>of());
```

Comment:
Why always add a `round_robin` policy into child policy? Even if its child policy is not empty.

Reply:
round_robin is the default policy, if everything in the list is not supported, it will use round_robin instead of error.

Reply:
Really? That's not the case for the existing implementation.

Reply:
The refactor design doc says so. The existing implementation used hard coded round_robin.